### DISCUSSION TITLE
Please sign-in problem

### DISCUSSION THREAD

**Discussion by ghost** [community/community#114502] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

When I use the github copilot chat and ask something I get an error.
Normally the autocomplete code works fine for me.
Who is facing this problem?
![Ekran görüntüsü 2024-03-27 201732](https://github.com/community/community/assets/56079439/a5e8f680-cfcb-416a-90a4-1f322d9cecab)

**Comment by Akash1134** [comment]
Hi @Ramil93 ,

You haven't specified your IDE in the issue. Assuming that you are using VSCode as an example, could you please try the following steps and let us know how it goes:
 

- In the bottom left corner of the Visual Studio Code window, click the Accounts icon, hover over your GitHub username, and click the Sign out button.
- To reload Visual Studio Code, press F1 to open the command palette, and select Developer: Reload Window.
- After Visual Studio Code reloads, sign back in to your GitHub account.

Thanks!

**Reply by ghost** [reply]
I use PHPStorm.
What should I do?

**Comment by davecheney** [comment, marked as answer]
@Ramil93 there is no copilot licence associated with this github account. you can sign up for a trial of github copilot here https://github.com/github-copilot/signup
